test/helpers/config.php

```diff
@@ -82,4 +82,4 @@
 });
 
 error_reporting(E_ALL | E_STRICT);
-?>
\ No newline at end of file
+?>
```

Comment:
In this case editorconfig is unrelated, because this wouldn't have been edited anyway :) Even though if we would add a editorconfig, it would be set to insert a newline, which then will gradually roll out through. So yeah I'm pro-editorconfig.